CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### NI-DCPower
     * #### Added
+        * `serial number` attribute
+        * `actual power allocation` attribute
+        * `requested power allocation` attribute
+        * `power allocation mode` attribute
```

Comment:
Combine these into single bullet (you can have multiple sub-bullets if you want).

Use "property" instead of attribute. That is the term used in Python world.